.github/workflows/awsfulltest.yml

```
@@ -15,7 +15,7 @@ jobs:
     steps:
       - name: Launch workflow via tower
         uses: nf-core/tower-action@v3
-        # TODO nf-core: You can customise AWS full pipeline tests as required
+        # nf-core: You can customise AWS full pipeline tests as required
```

```suggestion
        # TODO nf-core: You can customise AWS full pipeline tests as required
```
This still hasn't been done with bases2fastq and the Illumina software.